chef/cookbooks/neutron/templates/default/neutron.conf.erb

```diff
@@ -849,6 +849,13 @@ pool_timeout = <%= @sql_pool_timeout %>
 # PEM encoded client certificate cert file
 # certfile =
 
+<% if node.roles.include?("neutron-server") -%>
```

Comment:
and the role for the neutron-agents is anything else than neutron-server. Thats good.
Are we 100% sure, that this nova notifications are really only be sent from neutron-server and not from the agents or the computes? Is that somewhere documented?